bedrock-devnet/devnet/__init__.py

```diff
@@ -332,10 +336,10 @@ def devnet_test(paths):
     # Run the two commands with different signers, so the ethereum nonce management does not conflict
     # And do not use devnet system addresses, to avoid breaking fee-estimation or nonce values.
     run_commands([
-        CommandPreset('erc20-test',
-          ['npx', 'hardhat',  'deposit-erc20', '--network',  'devnetL1',
-           '--l1-contracts-json-path', paths.addresses_json_path, '--signer-index', '14'],
-          cwd=paths.tasks_dir, timeout=8*60),
+        # CommandPreset('erc20-test',
+        #   ['npx', 'hardhat',  'deposit-erc20', '--network',  'devnetL1',
+        #    '--l1-contracts-json-path', paths.addresses_json_path, '--signer-index', '14'],
+        #   cwd=paths.tasks_dir, timeout=8*60)
```

Comment:
Didn't mean to commit this diff. If reviewers are happy with the op-e2e devnet test, I will just remove this altogether. Can then remove the `devnet-tasks` in a follow up.

Reply:
I would like to remove these devnet tasks, perhaps its a better idea to see them pass in CI against the new devnet before removing them

Reply:
@tynes They don't pass and I cannot get them to work. I wasted a lot of time on this. This is the reason I switched to Go tests.